pkg/minikube/node/config.go

```diff
@@ -51,6 +51,11 @@ func showVersionInfo(k8sVersion string, cr cruntime.Manager) {
 	}
 }
 
+func showNoK8sVersionInfo(cr cruntime.Manager) {
+	version, _ := cr.Version()
```

Comment:
I don't think we should be ignoring this error, at the least we should be logging a `klog.Warningf` so we have it logged somewhere that that command failed.

Reply:
I've made the changes to capture the error and added a `klog.Warningf` if it returns an error.